lib/docker-toolbelt.coffee

```diff
@@ -62,6 +62,11 @@ getCacheId = Promise.method (dkroot, driver, layerId) ->
 getRandomFileName = (imageId) ->
 	"tmp-#{imageId.split(':')[1]}-#{randomstring.generate(8)}"
 
+# Check if the docker version is a release after 1.10.0, or if its one of the fun
+# new non-semver versions, which we incidentally know all appeared after 1.10.0
+isOldDockerVersion = (version) ->
```

Comment:
The comment and function name are incorrect. There were actually more semver-compliant Docker releases after 1.10 (I think up to 1.13). The check tests for that specific version because 1.10 changed the way images are stored on disk and referenced (they moved to what's called "content-addressable" layer format), so `isOldDockerVersion` isn't actually accurate. Maybe something like `supportsNewContentAddressableFormat` or something shorter maybe.

Reply:
Thanks for clarifying, I'll update the name and the comment 👍